src/main/java/com/eliasnogueira/driver/TargetFactory.java

```diff
@@ -27,16 +27,19 @@
 import com.eliasnogueira.enums.Target;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.Capabilities;
 import org.openqa.selenium.MutableCapabilities;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.edge.EdgeOptions;
+import org.openqa.selenium.firefox.FirefoxOptions;
 import org.openqa.selenium.remote.RemoteWebDriver;
 import org.testcontainers.containers.BrowserWebDriverContainer;
 
 import java.net.URL;
 
 import static com.eliasnogueira.config.ConfigurationManager.configuration;
-import static com.eliasnogueira.driver.BrowserFactory.CHROME;
-import static com.eliasnogueira.driver.BrowserFactory.valueOf;
+import static com.eliasnogueira.driver.BrowserFactory.*;
```

Comment:
Could you remove the start import and add the specialized ones?

Reply:
done